pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -120,6 +120,12 @@ func (p *provider) getConfig(s v1alpha1.ProviderConfig) (*Config, *providerconfi
 	if err != nil {
 		return nil, nil, nil, fmt.Errorf("failed to get the value of \"password\" field, error = %v", err)
 	}
+	// Ignore Region not found as Region might not be found and we can default it later
+	c.Region, err = p.configVarResolver.GetConfigVarStringValueOrEnv(rawConfig.Region, "OS_REGION_NAME")
+        if err != nil {
+		glog.V(0).Infof("Region from configuration or environment variable not found")
```

Comment:
Can you please make this a `glog.V(6).Infof` as its debug log? Other than that, pull LGTM

Reply:
ok, will do